hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -42,3 +42,45 @@ export interface Category {
     max_per_team: number;
     unique_hardware_count: number;
 }
+
+/** Event API */
```

Comment:
just to be consistent. add one more star lolllll.

Reply:
All of the ones above only have one closing star, so this is fine. No idea why, but it's normal to start a block comment like this with a double star :man_shrugging:.